src/bin/syntax.js

```diff
@@ -554,7 +554,10 @@ function getGrammar(grammarFile, mode) {
     return null;
   }
 
-  const grammarData = Grammar.dataFromGrammarFile(grammarFile, 'bnf');
+  const grammarData = Grammar.dataFromGrammarFile(grammarFile, {
```

Comment:
We'll need to update other calls to this new format:

https://github.com/DmitrySoshnikov/syntax/blob/8a029206c89eb046c7fc435496920bad00357aea/src/bin/syntax.js#L587